connector/kafka-0-10-sql/src/test/resources/error/kafka-error-classes.json

```diff
@@ -0,0 +1,26 @@
+{
+  "TOPIC_PARTITIONS_IN_END_OFFSET_ARE_NOT_SAME_WITH_PREFETCHED" : {
+    "message" : [
+      "Kafka data source in Trigger.AvailableNow should provide the same topic partitions in pre-fetched offset to end offset for each microbatch. ",
```

Comment:
This should be a transient issue right?  As in when the job is triggered again, the job should run fine right and number the job will read from whichever partitions exist?  Can you add that to this error message?

Reply:
We are guarding against arbitrary bugs so there is no 100% guarantee that this would only happen transiently, but I agree that restarting the query would mitigate the issue in most cases.